package.json

```diff
@@ -1830,7 +1830,7 @@
             "panel": [
                 {
                     "id": "jupyter-variables",
-                    "title": "",
+                    "title": "Jupyter",
```

Comment:
Seems like more people were seeing the leading ":" and some worry about having the unnamed container, so reverting.